models.go

```diff
@@ -136,6 +134,40 @@ type respUserGet struct {
 	// TODO: extattr external_profile external_position
 }
 
+// respUserGet 读取成员响应
+type respUserGet struct {
+	respCommon
+
+	respUserDetail
+}
+
+// reqUserByDept 部门成员请求
+type reqUsersByDeptID struct {
+	DeptID     int64
+	FetchChild bool
+}
+
+var _ urlValuer = reqUsersByDeptID{}
+
+func (x reqUsersByDeptID) intoURLValues() url.Values {
+	var fetchChild int64
+	if x.FetchChild {
+		fetchChild = 1
+	}
+
+	return url.Values{
+		"department_id": {strconv.FormatInt(x.DeptID, 10)},
+		"fetch_child":   {strconv.FormatInt(fetchChild, 10)},
+	}
+}
+
+// respUserByDept 部门成员详情响应
+type respUsersByDeptID struct {
+	respCommon
+
+	users []*respUserDetail `json:"userlist"`
```

Comment:
I believe the field must be public for `encoding/json` to not ignore.

```suggestion
	Users []*respUserDetail `json:"userlist"`
```